docs/examples.md

```diff
@@ -0,0 +1,170 @@
+            }
+        else:
+            # You should return at least one valid block here.
+            # This returns a raw_html block containing the custom HTML tag.
+            # It's a way to handle content that for some reason could not be parsed.
+            return {
+                "type": "raw_html",
+                "value": str(soup),
+            }
+```
+
+Because the handler will add a custom HTML tag to the content the default action of the bleach_clean() filter needs to be altered.
+
+Specifically, it needs to know what the custom HTML tag name is and what are valid attributes of the tag.
+
+Then generated custom HTML tag for the example above would be:
+
+```html
+<wagtail_block_anchor_image 
+    data-href="http://www.example.com/some-content" 
+    data-class="alignleft" 
+    data-src="http;//www.example.com/image.jpg" 
+    data-caption="Image title">
+</wagtail_block_anchor_image>
+```
+
+Before the custom HTML tag above is converted to a Wagtail block all the content is run through bleach_clean() to remove any unwanted HTML tags. The default list of HTML tags and attributes doesn't include the custom HTML here.
+
+### Add extra configuration to allow this HTML tag is pass through bleach_clean()
+
+In your own settings add the following:
+
+```python
+WAGTAIL_WORDPRESS_IMPORT_PREFILTERS = [
+    {
+        "FUNCTION": "wagtail_wordpress_import.prefilters.linebreaks_wp",
+    },
+    {
+        "FUNCTION": "wagtail_wordpress_import.prefilters.transform_shortcodes",
+    },
+    {
+        "FUNCTION": "wagtail_wordpress_import.prefilters.transform_inline_styles",
+    },
+    {
+        "FUNCTION": "wagtail_wordpress_import.prefilters.bleach_clean",
+        "OPTIONS": {
+            "ADDITIONAL_ALLOWED_ATTRIBUTES": {
+                "wagtail_block_anchor_image": "data-caption",
+                "wagtail_block_anchor_image": "data-class",
+                "wagtail_block_anchor_image": "data-href",
+                "wagtail_block_anchor_image": "data-src",
+            },
+        },
+    },
+]
+```
+
+The four `FUNCTIONS` are required to run the full list of prefilters and is copied here form the defaults in the package.
+
+The specific `OPTIONS` for `ADDITIONAL_ALLOWED_ATTRIBUTES` are required for this example and will be added to the configuration of `wagtail_wordpress_import.prefilters.bleach_clean` so the custom HTML tag with those attributes will pass through bleach_clean() and be available to the block builder when it creates a ImageBlock()
+
+### Loading your handler
+
+If you handler is in a module thats not loaded during the import, you can add it to your apps.py `ready()` method:
+
+```python
+# pages.apps.py
+
+from django.apps import AppConfig
+
+
+class PagesConfig(AppConfig):
+    name = "pages"
+
+    def ready(self):
+        from . import anchor_image_handler
+```
+
+The `@register` decorator for your handler will be called and the handler will be added to the list of handlers to parse the HTML content.
```

Comment:
**Comment**: While this information is useful, it's more related to how to use Django, than any code our package provides.